src/controllers/movies.js

```diff
@@ -0,0 +1,63 @@
+const { Prisma } = require('@prisma/client')
+const prisma = require('../utils/prisma')
+
+const getMovieList = async (req, res) => {
+	const movies = await prisma.movie.findMany({
+		orderBy: {
+			id: 'asc',
+		},
+		include: {
+			screenings: true,
+		},
+	})
+	return res.status(200).json({ movies: movies })
+}
+
+const createMovie = async (req, res) => {
+	const body = req.body
+	const newMovie = await prisma.movie.create({
+		data: {
+			title: body.title,
+			runtimeMins: body.runtimeMins,
+		},
+		include: {
+			screenings: true,
+		},
+	})
+	res.status(201).json({ movie: newMovie })
+}
+
+// by id
+const getById = async (req, res) => {
+	const id = Number(req.params.id)
+	const newMovie = await prisma.movie.findUnique({
```

Comment:
newMovie on finding by id could be confusing?

Reply:
Yeah I can see that, I'll modify that